openedx/features/survey_report/context_processors.py

```diff
@@ -0,0 +1,33 @@
+"""
+This is the survey report contex_processor modules
+
+This is meant to determine the visibility of the survey report banner
+across all admin pages in case a survey report has not been generated
+
+"""
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta  # for months test
+from .models import SurveyReport
+from django.urls import reverse
+
+
+def admin_extra_context(request):
+    """
+    This function sends extra context to every admin site
+
+    The current treshhold to show the banner is one month but this can be redefined in the future
+
+    """
+    months = 6
```

Comment:
Thanks for the change, but this is still a constant. I meant configuring this so that it reads from a variable. Similar to what [happens here](https://github.com/openedx/edx-platform/blob/d7b7f7070f671cbbc12e9630281720da068ffd6c/lms/envs/production.py#L1129).